versions/draft-3/SPEC.md

```diff
@@ -2359,6 +2359,12 @@ task example {
 Supported units are KiloByte ("K", "KB"), MegaByte ("M", "MB"), GigaByte ("G", "GB"), TeraByte ("T", "TB") as well as their [binary version](https://en.wikipedia.org/wiki/Binary_prefix) "Ki" ("KiB"), "Mi" ("MiB"), "Gi" ("GiB"), "Ti" ("TiB").
 Default unit is Bytes ("B").
 
+### Acceptable compound input types
+Varieties of the `size` function also exist for the following compound types. The `String` unit is always treated the same as above. Note that to avoid numberical overflow, very long arrays of files should probably favor larger units.
```

Comment:
Yikes having the default unit be bytes will undoubtedly...byte someone for "numberical" overflow

Reply:
just got bitten. @mcovarr ^^